### DISCUSSION TITLE
Styling of TextBoxes, Buttons, PasswordBoxes, etc. default to the "System Default" when developing WASM App

### DISCUSSION THREAD

**Discussion by DylanSafeAt** [unoplatform/uno#15386] [Q&A]
### Current behavior

Certain XAML Elements default styling changes based on the device thats running it. For example, on my windows machine, our login screen's email and password fields look like this (my windows machine's style is Dark Mode): 
![image](https://github.com/unoplatform/uno/assets/88735034/9ca56467-70ad-4dea-ba68-abf44bcfdaef)
But when I turn it to Light Mode, or try it on a Mac using any browser besides chrome, the fields look like this:
<img width="1390" alt="Screenshot 2024-02-06 at 1 30 09 PM" src="https://github.com/unoplatform/uno/assets/88735034/8e00f72a-87f9-4389-8eb0-b0cb517a203c">

Is there some global setting I can turn off in XAML that forbids defaulting to system style? Or am I barking up the wrong tree entirely?

Thanks for any help!

**Comment by jeromelaban** [comment]
Thanks for the question. 

You can force the theme used by the app by setting the `RequestedTheme="Light"` property in the `AppHead.xaml` file, or using `RequestedTheme = ApplicationTheme.Light;` in the `AppHead` constructor.
